ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -51,15 +56,18 @@
         void onActiveCardChanged(@Nullable View activeCard);
     }
 
-    private final GestureDetectorCompat mGestureDetector;
+    private Boolean mShouldAnimateCard = true;
```

Comment:
you should use the primitive `boolean` and not the boxed `Boolean`

Reply:
also, I would initialize this to `false`, default behavior shouldn't be to bounce.

Reply:
and rename this to `mBounceFirstCard`.

`mShouldAnimateCard` doesn't indicate if this boolean is about the bouncing animation or the swiping animation.